README.adoc

```diff
@@ -67,11 +67,11 @@ Just has a ton of useful features, and many improvements over Make:
   link:https://github.com/casey/just#writing-recipes-in-other-languages[arbitrary
   languages], like Python or NodeJS.
 
-- `just` can be invoked from any subdirectory, not just the directory that contains the `Justfile`.
+- `just` can be invoked from any subdirectory, not just the directory that contains the `justfile`.
 
 - And link:https://github.com/casey/just#manual[much more]!
 
-If you need help with `just` please feel free to open an issue or ping me on link:https://discord.gg/ezYScXR[discord]. Feature requests and bug reports are always welcome!
+If you need help with Just please feel free to open an issue or ping me on link:https://discord.gg/ezYScXR[Discord]. Feature requests and bug reports are always welcome!
```

Comment:
Here and later on, I think I actually prefer "`just`" instead of "Just". "Just" is such a common english word that I think it's good to use backticks to set it off.

Reply:
👍 Done